### DISCUSSION TITLE
Differences between PrismLauncher and MultiMC

### DISCUSSION THREAD

**Discussion by lmtr0** [PrismLauncher/PrismLauncher#654] [Q&A] [answered]
Hello there! Good day everybody!
I'm curious about the differences between PrismLauncher and MultiMC. I've tried searching about it, but only found this about Poly https://www.reddit.com/r/PolyMCLauncher/comments/unhud9/what_is_the_diference_beetwen_multimc_and_polymc/ also this https://www.reddit.com/r/PolyMCLauncher/comments/unhud9/comment/isqifwf/?utm_source=share&utm_medium=web2x&context=3 ._.

**Comment by DioEgizio** [comment]
There are a ton of more features in prism, but yeah maybe we should make a wiki page for showcasing some to explain the difference between prism and multimc

**Comment by TayouVR** [comment, marked as answer]
in a nutshell:
- mod browsing and installing from modrinth and curseforge right in launcher
- mod updating
- modpack updating
- tons of small bug fixes and QoL changes
- open packaging policies and lots of official and unofficial package sources (pretty much all linux package managers, homebrew, scoop, choco, winget.... to list a few)
- more modern codebase with e.g. Qt6 builds, which among other things handle fractional scaling correctly.

**Reply by DioEgizio** [reply]
Also curseforge support, redesigned instance icons and UI/UX improvements

**Reply by DioEgizio** [reply]
oh and also arm support

**Reply by lmtr0** [reply]
thank you :)

**Comment by MbtMarco** [comment]


**Reply by lmtr0** [reply]
makes sense

**Reply by lmtr0** [reply]
I was just curious
